app/src/main/res/values/strings.xml

```diff
@@ -53,4 +53,6 @@
         Haven't found some strings? There's more at
         https://github.com/SimpleMobileTools/Simple-Commons/tree/master/commons/src/main/res
     -->
+
+    <string name="start_stopwatch">Start Stopwatch</string>
```

Comment:
you have to add this at the proper place and in all languages

Reply:
can we add github bot services like translate wiki for this project it will auto translate for all language what do you think ?

Reply:
https://translatewiki.net/

Reply:
no, i dont want any bots. Lets just add the english version to all language files.

Reply:
ok sure.

Reply:
done